providers/os/resources/packages/windows_packages.go

```diff
@@ -129,7 +129,15 @@ func (p winAppxPackages) toPackage(platform *inventory.Platform) Package {
 		Arch:    p.arch,
 		Format:  "windows/appx",
 		Vendor:  p.Publisher,
-		PUrl:    purl.NewPackageURL(platform, purl.TypeAppx, p.Name, p.Version).String(),
+	}
+
+	// TODO: We need to figure out why we have empty displayNames.
+	// this is common in windows but we need to verify it is a windows
+	// issue and not a cnquery issue.
+	if p.Name != "" {
+		pkg.PUrl = purl.NewPackageURL(platform, purl.TypeAppx, p.Name, p.Version).String()
```

Comment:
The package will remain on the list, only without a PURL.

The other part of the PR skips the package completely. I think we should do the same with this.